rules-tests/DeadCode/Rector/Expression/RemoveDeadStmtRector/Fixture/skip_has_get_magic.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\Expression\RemoveDeadStmtRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+use Rector\Tests\DeadCode\Rector\Expression\RemoveDeadStmtRector\Source\Entity;
+
+class SkipHasGetMagic extends TestCase
+{
+    public function testThrow()
+    {
+        $obj = new Entity();
+        $this->expectException(InvalidArgumentException::class);
+
+        $obj->invalid;
```

Comment:
when reading it, this comes to mind: not sure how this rector works on `$obj->invalid;()` aka magic-`__call`.

maybe thats another relevant test-case?

Reply:
that's seems already handled, I tried that:

![Screen Shot 2021-08-07 at 16 46 51](https://user-images.githubusercontent.com/459648/128596159-ce9d8d73-e434-4fee-9a6a-c071380cd145.png)